docs/proposals/timeseries-partitioning-in-compactor.md

```diff
@@ -264,4 +295,128 @@ T2 partition 4 - Hash(timeseries label) % 4 == 3 && % 8 == 7
 T3 partition 8 - Hash(timeseries label) % 8 == 7
 ```
 
+### Compaction Partitioning Examples
+
+#### Ideal case:
```

Comment:
nit: I think it's better if the title is like `All source blocks were compacted by partitioning compaction (Idea case)`